src/no-lodash-constant.js

```diff
@@ -4,8 +4,8 @@ module.exports = {
   meta: {
     fixable: 'code'
   },
-  create: ctx => ({
-    CallExpression(node) {
+  create: (ctx) => ({
```

Comment:
we set prettier to avoid parens - what happened here?

Reply:
Since we are not installing the eslint-config-smartprocure package any longer to avoid a circular dependency, I've copied over our rules from there. Added an eslint rule for this particular rule (among others).

We should let eslint to do the things eslint does best and let prettier do the spooky-at-a-distance stuff.